quickstarts/deploy.markdown

```diff
@@ -0,0 +1,54 @@
+Kroxylicious is a Java application based on [Netty](https://netty.io/), which means it will run anywhere you can run a JVM. (That's a lot of places!)
+To keep things simple, we've created a quick setup guide to help.
+
+<br />
+
+## Getting started
+
+#### Prerequisites
```

Comment:
Isn't a Kafka cluster running on the default port also a pre-requirement?

Reply:
I guess a running cluster is, but the port just needs to be in sync with the Kroxylicious config file.

Reply:
I had made the assumption that if someone was looking at setting up their own Kroxylicious instance that they would already have a Kafka cluster running (why else would they want to do this?), but maybe it's worth calling it out specifically.

Reply:
I don't think we need to go any further than saying we need a cluster running. Can always point people to our friends Strimzi for help getting a cluster going...